lib/__tests__/KTable.spec.js

```diff
@@ -0,0 +1,155 @@
+import { mount } from '@vue/test-utils';
+import KTable from '../KTable';
+
+describe('KTable.vue', () => {
+  it('should mount the component', () => {
+    const headers = [
+      { label: 'Name', dataType: 'string' },
+      { label: 'Age', dataType: 'numeric' },
+      { label: 'City', dataType: 'string' },
+    ];
+    const rows = [
+      ['Alice', 25, 'New York'],
+      ['Bob', 30, 'Los Angeles'],
+      ['Charlie', 35, 'San Francisco'],
+    ];
+    const wrapper = mount(KTable, {
+      propsData: {
+        headers,
+        rows,
+        useLocalSorting: true,
```

Comment:
`useLocalSorting` is the obsolete API, so this place will need to be updated

Reply:
done